.gitlab-ci.yml

```diff
@@ -21,13 +21,17 @@ cache:
 
 .releaseable_branches:             # list of git refs for building GitLab artifacts (think "pre-release binaries")
   only:                            &releaseable_branches
-    - stable
-    - beta
-    - tags
+    refs:
+      - stable
+      - beta
+      - tags
+    variables:
+      - $SCHEDULE_TAG == "nightly"
+
 
 .collect_artifacts:                &collect_artifacts
   artifacts:
-    name:                          "${CI_JOB_NAME}_${CI_COMMIT_REF_NAME}"
+    name:                          "${CI_JOB_NAME}_${SCHEDULE_TAG:-${CI_COMMIT_REF_NAME}}"
```

Comment:
what happens if schedule_tag is empty?

Reply:
this is posix parameter expansion ':-' does return the value of the variable $SCHEDULE_TAG if set, otherwise the value of $CI_COMMIT_REF_NAME} will be given.